actors/common/src/main/java/org/sunbird/learner/actors/BackgroundJobManager.java

```diff
@@ -40,6 +40,7 @@
 @ActorConfig(
   tasks = {},
   asyncTasks = {
+          "mergerUserSToElastic",
```

Comment:
mergerUserSToElastic to mergerUserToElastic

Reply:
done